tests/lms/util/test_authorize_lms.py

```diff
@@ -65,20 +64,15 @@ def test_it_redirects_for_oauth(self, lti_launch_request):
         assert response.code == 302
         assert "https://hypothesis.instructure.com/login/oauth2/auth" in response.location
 
-    def test_it_saves_state(self, lti_launch_request):
+    def test_it_saves_lti_params(self, lti_launch_request):
```

Comment:
Is there any value in being explicit that it "saves" them in the database (i.e. that the persistence layer is the DB) in this test name?